docs/source/daml/code-snippets/Reference.daml

```diff
@@ -46,23 +46,23 @@ template NameOfTemplate
 -- start template choice snippet
     -- option 1 for specifying choices: choice name first
     choice NameOfChoice1
-          : ()  -- replace () with the actual return type
-        with
-          exampleParameter : Text -- parameters here
+      : ()  -- replace () with the actual return type
```

Comment:
the idea behind this indentation is to signal that each of
1. choice return type
2. choice argument fields
3. choice controller
4. choice body

can be considered as components of the choice declaration at the same level, each introduced with its own keyword (or `:`) (compare with the previous version where some choice bodies (`do` blocks) appear to be nested under the `controller` declarations).